### PR TITLE
Validate exported to wrong namespace config details

### PR DESCRIPTION
**Pull request by hhovsepy**
Issue https://github.com/kiali/kiali/issues/7061

Create a VS, DR, SE with wrong 'exportTo' namespace:
```apiVersion: networking.istio.io/v1alpha3
kind: ServiceEntry
metadata:
  name: www.googleapis.com
spec:
  hosts:
    - www.googleapis.com
  ports:
    - number: 80
      name: http-port
      protocol: HTTP
    - number: 443
      name: http-port-for-tls-origination
      protocol: HTTP
  resolution: DNS
  exportTo:
    - wrong
---
apiVersion: networking.istio.io/v1alpha3
kind: VirtualService
metadata:
  name: www.googleapis.com
spec:
  hosts:
    - www.googleapis.com
  http:
    - match:
        - port: 80
      route:
        - destination:
            host: www.googleapis.com
            port:
              number: 443
  exportTo:
    - wrong
---
apiVersion: networking.istio.io/v1alpha3
kind: DestinationRule
metadata:
  name: www.googleapis.com
spec:
  host: www.googleapis.com
  trafficPolicy:
    loadBalancer:
      simple: ROUND_ROBIN
    portLevelSettings:
      - port:
          number: 443
        tls:
          mode: SIMPLE # initiates HTTPS when accessing www.googleapis.com
  exportTo:
    - wrong
```
Check Istio Config List, you will see the errors there for those objects.
Screenshot from 2024-01-29 18-26-15
Check the details page, errors should be there for 'wrong' namespace exported. Also all other validation messages should be shown if they exists:
Screenshot from 2024-01-29 18-27-36

### CODE REVIEW

**Review by josunect — COMMENTED** [review]
Works as expected: 

image

image

Just a minor comment in the code.

File: handlers/istio_config.go [hunk 1]
```diff
@@ -81,8 +81,22 @@ func IstioConfigList(w http.ResponseWriter, r *http.Request) {
...
 			defer wg.Done()
-			// We don't filter by objects when calling validations, because certain validations require fetching all types to get the correct errors
-			istioConfigValidationResults, errValidations := business.Validations.GetValidations(context.TODO(), cluster, namespace, "", "")
+			istioConfigValidationResults := models.IstioValidations{}
+			var errValidations error
+			// We don't filter by service and workload when calling validations, because certain validations require fetching all types to get the correct errors
+			// when namespace is empty, validaions should be done per all namespaces to apply object filters
+			if namespace == "" && len(nss) > 0 {
+				for _, ns := range nss {
+					nsValidations, nsErr := business.Validations.GetValidations(context.TODO(), cluster, ns, "", "")
```
[the remark this answers is not in the document]
Comment (hhovsepy) [reply handlers/istio_config.go]: right, good catch

**Review by josunect — APPROVED** [review]
LGTM

**Comment by hhovsepy** [comment]
thank you @josunect

**Pull request merged**